sherlock/sherlock.py

```diff
@@ -556,6 +556,10 @@ def main():
                         action="store_true", default=False,
                         help="Force the use of the local data.json file.")
 
+    parser.add_argument("--nsfw", "-n",
```

Comment:
Remove `-n`, its enough with `--nsfw` in my opinion. Because a user may accidentally type in `-n` when meaning to use `-l`. So having the user to type out `--nsfw` will become a sort of proper confirmation if that makes sense